tracer/test/Datadog.Trace.TestHelpers/SpanTagAssertion.cs

```diff
@@ -24,9 +24,10 @@ internal SpanTagAssertion(Result result, IDictionary<string, T> tags)
         public static void DefaultTagAssertions(SpanTagAssertion<T> s) => s
             .IsPresent("env")
             .IsOptional("runtime-id") // TODO: Make runtime-id required on all spans, per our span attributes push
-            .IsOptional("language") // TODO: Make language required on all spans, per our span attributes push
+            .IsOptional("language")   // TODO: Make language required on all spans, per our span attributes push
             .IsOptional("version")
-            .IsOptional("_dd.p.dm")
+            .IsOptional(Tags.Propagated.DecisionMaker) // _dd.p.dm, aka sampling mechanism
```

Comment:
nit: Can these be the strings instead of the constants? There was some discussion about it on another thread for the git ones where I think this was the conclusion

Reply:
Yup, I started that other discussion after I added this 😅 Thanks for reminding me!

Fixed.